src/python/pants_release/generate_github_workflows.py

```diff
@@ -316,6 +316,17 @@ def install_go() -> Step:
     }
 
 
+def install_protoc() -> Step:
+    return {
+        "name": "Install Protoc",
+        "uses": "arduino/setup-protoc@9b1ee5b22b0a3f1feb8c2ff99b32c89b3c3191e9",
+        "with": {
+            "version": "23.x",
+            "repo-token": "${{ secrets.GITHUB_TOKEN }}",
```

Comment:
I'm not keen on this, but it sounds like it's necessary in practice due to those rate limits. Which I find surprising, but...

Looking at that action, it's very annoying to me that it does the version scraping even if you pin to a specific protoc version. In general it seems like we don't need 95% of what that action does (resolve protoc version semvers), and writing our own action to just download the protoc we want would be like 10 lines of code?

Reply:
I mean, I think this is fine for now. Arduino are a trustworthy project that are less likely than average to be compromised, I hope. But passing a token to a third-party action still gives me pause.

Reply:
Probably would be like 10 lines of code. We mainly want the action to download protoc into the GitHub Actions tool cache which is a specific directory on the machine.

Reply:
I can try adding something to https://github.com/pantsbuild/actions.

Reply:
Actually, easier would be to just fork that repo to the `pantsbuild` org.

Reply:
I don't think we need to do that, since we're pinning to a specific SHA of that repo. It would just be a concern when upgrading it.

Reply:
Then I should probably comment the reference in the generator script to note that any upgrade will need source code review. Thoughts?

Reply:
Yeah, maybe just mention that since we're passing it the token we should be careful when upgrading.